conf/adcm/python_scripts/run_ldap_sync.py

```diff
@@ -198,6 +217,25 @@ def _sync_ldap_users(self, ldap_users):
         msg += f"Couldn\'t synchronize users: {error_names}\n" if error_names else ""
         log.debug(msg)
 
+    def __process_user_ldap_groups(self, user, user_dn):
```

Comment:
Maybe use protected method `def _method` as soon as private is not really necessary (we can discuss it with team)

Reply:
done